src/scss/components/_window-switcher.scss

```diff
@@ -24,6 +24,8 @@
 .window-switcher__input{
   &:focus{
     outline: 1px solid $orange;
+    padding-left: 4px;
+    padding-right: 4px;
```

Comment:
👍

Reply:
A QQ about this though.. were you able to achieve this without the UI shifting when going into edit mode? If not, that would be something to achieve, keeping the UI from shifting by a few pixels when going into edit mode. That's the reason why I used outline as opposed to borders, and also why I haven't looked into it since then 😅

Reply:
To achieve that would prolly mean having the padding there by default (with or without focus)

Reply:
Good point. We should do that then.